### PR TITLE
Merge master-vs-deps into master

### PR DESCRIPTION
**Pull request by jasonmalinowski**
Yes, this is the backwards merge, but @ivanbasov and @dpoeschl it doesn't look like anything in this payload has any dependency on a newer build of Visual Studio. Any reason we can't put this stuff back in master?

### CODE REVIEW

**Comment by ivanbasov** [comment]
1. It is necessary to use WpfTheory not WpfFact for unit tests the same way as we do for other test scenarios in the file.
2. master already contains those changes. Therefore, this can be submitted into master. master-vs-deps is not requires.

**Comment by jasonmalinowski** [comment]
> master already contains those changes. Therefore, this can be submitted into master. master-vs-deps is not requires.

I'm confused. This is a PR _from_ master-vs-deps (where these changes originally where made) into master.

File: src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_InternalsVisibleTo.vb [hunk 1]
```diff
@@ -639,5 +639,265 @@ using System.Reflection;
                 Await state.AssertNoCompletionSession()
             End Using
         End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        <WorkItem(29447, "https://github.com/dotnet/roslyn/pull/29447")>
+        Public Async Function CodeCompletionReplacesExisitingAssemblyNameWithDots_1() As Task
```
Comment (sharwell) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_InternalsVisibleTo.vb]: @jasonmalinowski This method should use the same approach as the existing test located above it. Same with the other methods which appear added in this file.

Comment (jasonmalinowski) [reply src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_InternalsVisibleTo.vb]: Fixing, although @sharwell and @ivanbasov I'm confused. The state here where some things are using Theories and some are using Facts is the current state in master-vs-deps, and not something being brought by this merge. Do we have other tests still broken?

Comment (ivanbasov) [reply src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_InternalsVisibleTo.vb]: Good catch! It seems that those WpfFacts were added there after I migrated tests to WpfTheories. It seems I have to migrate those new tests too :-(

**Comment by sharwell** [comment]
@jasonmalinowski Build probably won't pass until we merge 16.0.x to master.

**Comment by sharwell** [comment]
retest this please

**Comment by jasonmalinowski** [comment]
@sharwell It did pass, want to take a look?

**Pull request merged**